lib/command/src/Obelisk/Command.hs

```diff
@@ -374,7 +371,14 @@ ob = \case
     DeployCommand_Update -> deployUpdate "."
     DeployCommand_Test (platform, extraArgs) -> deployMobile platform extraArgs
   ObCommand_Run -> run
-  ObCommand_Profile basePath -> profile basePath
+  ObCommand_Profile mBasePath -> do
+    basePath <- case mBasePath of
+      Just path -> pure path
+      Nothing -> do
+        time <- liftIO $ getCurrentTime
+        root <- findProjectRoot "."
+        pure $ (fromMaybe "" root) </> "profile" </> formatTime defaultTimeLocale "%Y-%m-%dT%H:%M:%S" time
```

Comment:
You can use `withProjecRoot` here to protect against the case when the root isn't found.